src/linter-jscs.js

```diff
@@ -97,6 +97,19 @@ export default class LinterJSCS {
         }
       });
     }));
+
+    this.subscriptions.add(atom.commands.add('atom-text-editor', {
+      'linter-jscs:fix-file': () => {
+        const textEditor = atom.workspace.getActiveTextEditor();
+
+        if (!textEditor || textEditor.isModified()) {
```

Comment:
Thanks for the feedback, wasn't aware of that! I just looked at the fix file command of the ESHint linter (I don't much experience with Atom). I'll add a new commit to fix this soon!